persistence/persistence-mybatis/src/test/java/org/cbioportal/persistence/mybatis/TreatmentMyBatisRepositoryTest.java

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright (c) 2016 Memorial Sloan-Kettering Cancer Center.
```

Comment:
The Hyve

Reply:
Changed to:
` * Copyright (c) 2019 The Hyve.`